src/main/java/Duke.java

```diff
@@ -1,10 +1,138 @@
+import duke.tasks.Task;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Todo;
+import duke.exceptions.EmptyDescriptionException;
+import duke.exceptions.TaskToMarkDoesNotExistException;
+import duke.exceptions.UnknownCommandException;
+
+import java.util.Scanner;
+
 public class Duke {
+    private static void displayList(Task[] tasks) {
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < Task.numberOfTasks; i += 1) {
+            System.out.print(i + 1 + ". ");
+            tasks[i].printTask();
+        }
+    }
+
+    private static void markTask(Task[] tasks, String task) {
+        tasks[Integer.parseInt(task) - 1].setDone();
+        System.out.println("Nice! I've marked this task as done:");
+        displayList(tasks);
+    }
+
+    private static void unmarkTask(Task[] tasks, String task) {
+        tasks[Integer.parseInt(task) - 1].setUndone();
+        System.out.println("OK, I've marked this task as not done yet:");
+        displayList(tasks);
+    }
+
+    private static void createTodo(Task[] tasks, String task) {
+        tasks[Task.numberOfTasks] = new Todo(task);
+        System.out.println("Got it. I've added this task:");
+        tasks[Task.numberOfTasks - 1].printTask();
+        System.out.println("Now you have " + Task.numberOfTasks + " tasks in your list.");
+    }
+
+    private static void createEvent(Task[] tasks, String task) {
+        String[] words = task.split("/from");
+        String description = words[0];
+        String[] words2 = words[1].split("/to");
+        String start = words2[0];
+        String end = words2[1];
+        tasks[Task.numberOfTasks] = new Event(description, start, end);
+        System.out.println("Got it. I've added this task:");
+        tasks[Task.numberOfTasks - 1].printTask();
+        System.out.println("Now you have " + Task.numberOfTasks + " tasks in your list.");
+    }
+
+    private static void createDeadline(Task[] tasks, String task) {
+        String[] words = task.split("/by");
+        String description = words[0];
+        System.out.println(words[0]);
+        System.out.println(words[1]);
+        String end = words[1];
+        tasks[Task.numberOfTasks] = new Deadline(description, end);
+        System.out.println("Got it. I've added this task:");
+        tasks[Task.numberOfTasks - 1].printTask();
+        System.out.println("Now you have " + Task.numberOfTasks + " tasks in your list.");
+    }
+
+    private static String[] getInput() {
+        Scanner input = new Scanner(System.in);
+        String text = input.nextLine(); // input the whole sentence into text
+        return text.split(" ", 2);
+    }
+
+    private static void editList() throws UnknownCommandException, EmptyDescriptionException, TaskToMarkDoesNotExistException {
+        String[] splitText = getInput();
+        Task[] tasks = new Task[100];
```

Comment:
Your current code structure only accepts a maximum of 100 tasks for now. Moving on, you might want to make this more flexible by using ArrayList.